### DISCUSSION TITLE
v8.5.0

### DISCUSSION THREAD

**Discussion by DanWBR** [DanWBR/dwsim#490] [General]
I'm really happy with this release because I was able to remove the dependency on the additional Python environments required to run Reaktoro and PEM Fuel Cell models. You'll only need to provide the path to a valid Python 3.7-3.9 installation and DWSIM will handle the rest.

Also, thanks to the anonymous analytics shared by most DWSIM users, I was able to know that one of the most used screen resolutions is 1366x768. On such screens, the location of the object palette is not optimal and ends up stealing useful space to work with the PFD. For this reason, I redesigned the palette and moved it to the right of the screen. Hope you like it. 😊

As a bonus, I included the Aqueous Electrolytes (Reaktoro) Property Package in the default set of models, so you don't need to download any additional packages to work with electrolytes anymore. Reaktoro by [Allan Leal](https://www.linkedin.com/in/ACoAAA1KUYwBAdw8AiJY6YHF-inNBDODLwRwdxE) is now a very important part of DWSIM. I'm still using Reaktoro v1.2.3, but I'll work hard to update it to v2.x in the near future. This will allow you to use newer Python versions like 3.10 and 3.11 with DWSIM.

**Changes since v8.4.7:**

- New Object Palette (Classic UI)
- Added Aqueous Electrolytes (Reaktoro) Property Package
- Added Cut/Copy/Paste items to context menus (Classic UI)
- Removed dependency of the additional Python environments package
- Fixed an issue with Gibbs Reactor
- Removed interactive Python console (Classic UI)
- Fixed compound search (Classic UI) (#489)
- Fixed Infinity Loop on Calc Critical Point in PR78 (#487)
- Other fixes and enhancements

**Full Changelog**: https://github.com/DanWBR/dwsim/compare/v8.4.7...v8.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/DanWBR/dwsim/releases/tag/v8.5.0'>v8.5.0</a>.</em>

**Comment by einhander** [comment]
> I redesigned the palette and moved it to the right of the screen. Hope you like it. 

+1 for 8.5.0! The ClassicUI for Linux looks a much better (although its not supported), and the no UI errors too!

**Comment by benicovdw** [comment]
Stupid question ... I can't get the .debs for ver 8.5/6/7 running on Ubuntu 22.04. 
========== SOLVED =================
Default FONTS in KDE control panel were set to Noto Sans ... which does not work with DWSIM.
Changed it to Sans ... and all is working fine again.


System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.TypeInitializationException: The type initializer for 'DWSIM.CrossPlatform.UI.Controls.ReoGrid.Rendering.StaticResources' threw an exception. ---> System.ArgumentOutOfRangeException: Font Family specified is not supported by this system
Parameter name: familyName
Actual value was Noto Sans._
